src/components/shared/loaders/loadingOverlay.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react'
+
+import SpinningSquares from './spinningSquares'
+
+import styles from './loadingOverlay.scss'
+
+type Props = {
+  delay?: number // milliseconds
+  className?: string
+}
+
+const STUPID_PHRASES = [
```

Comment:
Maybe we update the name of these before merging (lol)

Reply:
Nah nah, I like this name lol.